Content/Items/Misc/Accessories.AxeBook.cs

```diff
@@ -0,0 +1,541 @@
+﻿using StarlightRiver.Content.Items.BaseTypes;
+using StarlightRiver.Core.Systems.CameraSystem;
+using StarlightRiver.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria.GameContent;
+using Terraria.Graphics.Effects;
+
+namespace StarlightRiver.Content.Items.Misc
+{
+	internal class AxeBook : SmartAccessory
+	{
+		public int comboState;
+
+		public override string Texture => AssetDirectory.MiscItem + "AxeBook";
+
+		public AxeBook() : base("Tiger Technique", "Allows execution of combos with axes\nThe final strike will rend enemy armor\nRight click to throw your axe") { }
```

Comment:
We really need some kind of expanded tooltips,. When I see this I wonder if this rend effect is good. I'd want to know things like if this debuff stacks, how much it reduces the armor by (which we should probably stay consistent with vanilla and call it defense anyway) how long it lasts. Vanilla is also really bad about this, requiring you to go to the wiki to actually determine if an item is good but we could be better